.changeset/pink-pots-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ithaka/pharos': patch
```

Comment:
Since it's adding new functionality, probably a minor release?

>MINOR version when you add functionality in a backward compatible manner 
(https://semver.org/)

Reply:
Yeah, good point @brentswisher